src/utils/api_types.ts

```diff
@@ -241,3 +242,26 @@ export interface Database {
   createdAt: string;
   updatedAt: string;
 }
+
+export interface Cron {
+  cron_spec: {
+    name: string;
+    schedule: string;
+    backoff_schedule?: number;
+  };
+  status?: CronStatus;
+  history: CronExecutionRetry[][];
+}
+
+export interface CronExecutionRetry {
+  status: "success" | "failure" | "executing";
+  start_ms: number;
+  end_ms: number;
+  error_message?: string;
+  deployment_id: string;
+}
```

Comment:
Though not something actionable here, but casing inconsistency happens here 👀

Reply:
Yeah, that's a pitty. Cron API is still a bit rough.